cache/redis_client.go

```diff
@@ -185,10 +191,28 @@ func NewRedisClient(logger log.Logger, name string, config RedisClientConfig, re
 	}
 	if config.MaxGetMultiConcurrency > 0 {
 		c.getMultiGate = gate.New(
-			prometheus.WrapRegistererWithPrefix("redis_getmulti_", reg),
+			prometheus.WrapRegistererWithPrefix(getMultiMetricNamePrefix, reg),
 			config.MaxGetMultiConcurrency,
 		)
 	}
+
+	c.clientInfo = promauto.With(reg).NewGaugeFunc(prometheus.GaugeOpts{
+		Name: clientInfoMetricName,
```

Comment:
You were convincing about using per-backend specific metric names. Now we're using a common metric name. What has changed?

Reply:
Yes nothing changes. Initially I wanted to keep cache name in the metric name for client_info but now decided to just use backend label as everything else (backward compatible behaviour is kept for memcached_client_info). I just moved "client_info" to common variable name because it is used in two places now.

Reply:
Ok. Sounds good. Don't have a big preference between the two options.